bad_actors/flask_api/validate_json.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import print_function
+import sys
+import jsonschema
+from jsonschema import validate
+
+
+def validate_json(json, schema):
```

Comment:
I think a more clean way to do this is if the validate_json function would only get the json, and the scheme will be loaded from the config (instead of doing it in the flask_api.py and passing the scheme to the function)